pkg/nvkind/cluster.go

```diff
@@ -123,17 +134,53 @@ func (c *Cluster) Create(opts ...ClusterCreateOption) error {
 	return nil
 }
 
+func (c *Cluster) PrintClusterInfo() error {
+	// A note on `kubectl` invocations: not specifying `--context <name>` would
+	// propbably work in most of the cases because `kind` also sets the default
+	// context. However, it is better to be explicit. For concurrent executions
+	// of `nvkind` it is certainly a requirement to be explicit about the
+	// context to prevent race conditions.
+
+	// Display endpoints against which kubectl is configured.
+	cmd := []string{
+		"kubectl", "cluster-info", "--context", "kind-" + c.Name,
+	}
+	if err := runChildProcess(cmd, ""); err != nil {
+		return fmt.Errorf("running process: %w", err)
+	}
+
+	cmd = []string{
+		"kubectl", "--context", "kind-" + c.Name, "get", "nodes",
+	}
+	if err := runChildProcess(cmd, ""); err != nil {
+		return fmt.Errorf("running process: %w", err)
+	}
+
+	return nil
+}
+
+func (c *Cluster) RegisterNvidiaRuntimeClass() error {
+	// Register runtimeclass.node.k8s.io/nvidia, allowing usage of
+	// --set runtimeClassName=nvidia upon helm-installing e.g. the k8s-device-plugin.
+	cmd := []string{
+		"kubectl", "--context", "kind-" + c.Name, "apply", "-f", "-",
+	}
+
+	if err := runChildProcess(cmd, k8sRuntimeClassYaml); err != nil {
+		return fmt.Errorf("running process: %w", err)
+	}
+
+	return nil
+}
+
+// Note(JP): note exposed via CLI, we may want to remove this for now.
```

Comment:
```suggestion
// Note(JP): not exposed via CLI, we may want to remove this for now.
```

Reply:
or add it to the CLI ....

Reply:
I thought about that. Will do in a follow-up if we like that. I felt OK about using `kind delete ...` for now.